autopush/tests/__init__.py

```diff
@@ -18,17 +21,26 @@
 
 def setUp():
     logging.getLogger('boto').setLevel(logging.CRITICAL)
-    boto_path = os.path.join(root_dir, "automock", "boto.cfg")
-    boto.config.load_from_path(boto_path)
     global ddb_process
     cmd = " ".join([
         "java", "-Djava.library.path=%s" % ddb_lib_dir,
         "-jar", ddb_jar, "-sharedDb", "-inMemory"
     ])
+    conf = botocore.config.Config(
+        region_name=os.getenv('AWS_REGION_NAME', 'us-east-1')
+    )
     ddb_process = subprocess.Popen(cmd, shell=True, env=os.environ)
+    autopush.db.DYNAMODB = boto3.resource(
+        'dynamodb',
+        config=conf,
+        endpoint_url=os.getenv("AWS_LOCAL_DYNAMODB", "http://127.0.0.1:8000")
```

Comment:
I think we'll have to do something about credentials here (hardcode?) so tests run locally without needing to set them